docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:16.04
+
+# Update
+RUN apt-get update && apt-get dist-upgrade -y
+
+# Install the necessary packages.
+RUN apt-get install -y \
+    supervisor apache2 git php7.0 libapache2-mod-php7.0 php7.0-cli \
+    php7.0-mysql php7.0-xml php7.0-mbstring php7.0-curl php7.0-zip
+
+RUN a2enmod php7.0
+RUN a2enmod rewrite
+
+# Add the files to the container.
+RUN mkdir /var/www/polr
+COPY . /var/www/polr/
+
+# Remove the storage directories, our startup script and symlinking will handle this.
+RUN rm -rf /var/www/polr/storage
+
+# Put the apache configuration in place
+RUN mv /var/www/polr/docker/apache.conf /etc/apache2/sites-available/000-default.conf
+
+# Configure supervisor to keep apache running.
+RUN mv /var/www/polr/docker/supervisor.conf /etc/supervisor/conf.d/apache.conf
+
+# Install and run composer to add necessary packages. Personally, I would prefer it if vendor was
+# bundled with releases, and not have this section
+RUN apt-get install curl  -y
+RUN curl -sS https://getcomposer.org/installer | php
+RUN mv composer.phar /usr/bin/composer
+RUN chmod +x /usr/bin/composer
+WORKDIR /var/www/polr
+RUN composer install
```

Comment:
composer warns against running as root: https://getcomposer.org/doc/faqs/how-to-install-untrusted-packages-safely.md

Is there a way we can avoid this?